python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -274,6 +279,32 @@ def _update(self):
         for node_id in nodes:
             self.recover_if_needed(node_id, now)
 
+    def _keep_min_worker_of_node_type(self, node_id, node_type_counts):
+        """Returns if workers of node_type should be terminated.
+
+        Receives the counters of running nodes so far and determines if idle
+        node_id should be terminated or not. It also updates the counters.
+
+        Args:
+            node_type_counts(Dict[NodeType, int]): The non_terminated node
+                types counted so far.
+        Returns:
+            node_type_counts(Dict[NodeType, int]): The updated non_terminated
+                node types counted so far.
```

Comment:
This isn't a return, so should be moved to the description or removed.

Reply:
It is returned by reference.

Reply:
I think it's fine (or maybe preferable) to just leave it as an argument and note that it's an output argument.